File: challenges/2-idade-em-dias/swift/godsensehl/idade-em-dias.swift
let input = readLine()

var anos = 0
var meses = 0

if let idade = input {
    var result = Int(idade)!
    if result >= 365 {
        anos = Int(result / 365)
        result -= 365 * anos
    }
    print("\(anos) ano(s)")
    if result >= 30 {
        meses = Int(result / 30)
        result -= 30 * meses
    }
    print("\(meses) mes(es)")
    print("\(result) dia(s)")
}

File: challenges/2-idade-em-dias/python/mwives/idade-em-dias.py
idadeEmDias = int(input("Entre com a idade em dias: "))

DIAS_NO_ANO = 365
DIAS_NO_MES = 30

anos = idadeEmDias // DIAS_NO_ANO
meses = (idadeEmDias % DIAS_NO_ANO) // DIAS_NO_MES
dias = (idadeEmDias % DIAS_NO_ANO) % DIAS_NO_MES


print(anos, "ano(s)")
print(meses, "mes(es)")
print(dias, "dia(s)")